vm/src/stdlib/os.rs

```diff
@@ -168,6 +168,8 @@ pub(crate) fn fspath(
     check_for_nul: bool,
     vm: &VirtualMachine,
 ) -> PyResult<FsPath> {
+    // path_converter
+    // unicode -> bytes -> buffer -> index -> fspath(unicode/bytes) -> TypeError
```

Comment:
I'm not versed with `os` stuff but I see this as a pretty common utility function used in `CPython` for many of the functions in `os` (in the argument clinic generated files). So wouldn't it make more sense in `TryFromObject`?

Reply:
I agree